dags/process_elife_zip_dag.py

```diff
@@ -45,12 +42,12 @@
 }
 
 
-def is_allowed(file_name: str) -> bool:
-    return Path(file_name).suffix in ALLOWED_EXTENSIONS
-
-
-def is_image(file_name: str) -> bool:
-    return Path(file_name).suffix in IMAGE_EXTENSIONS
+def is_image(file: bytes) -> bool:
+    try:
+        Image.open(BytesIO(file))
```

Comment:
from [the documentation](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.open) it seems this method is lazy enough not to read the contents of the files in memory, but it states that there is a file descriptor being left open for later usage. Does it need a `close()` call then?

Reply:
Oh wait, the bytes are already in memory... so should they? If we are only identifying whether the file is an image or not we shouldn't need to read all of it right? e.g. It could be a 4GB video

Reply:
I think this makes sense from the context of a caller but less in a generic `is_image` function :thinking:

Reply:
`is_image` removed. Agreed to only convert tiff files.